src/source/reference/get-feed-import-id-detail.rst

```diff
@@ -1,9 +1,117 @@
 ============================
 
+:ref:`get_feed_import_id_detail_v2` | :ref:`get_feed_import_id_detail_v1`
+
+.. _get_feed_import_id_detail_v2:
+
+GET /feed/import/{id}/detail V2
+-------------------------------
+
+.. list-table::
+ :widths: 20 80
+
+ * - Scope
+   - ``api_ro`` or ``console_ro``
+
+ * - Accept
+   - ``application/sellside.feedimportdetail-v2+json, application/json``
+
+This endpoint provides a summary of the feed import. Use this endpoint to get information on whether there were any
+errors or warnings associated with a specific import.
+
+=============  ======= ====================================
+Field          Type    Description
+=============  ======= ====================================
+id             int     import ID, same as provided in the call.
+url            string  URL where the feed file was downloaded.
+status         string  status of the feed import. See :ref:`get_feed_import_id_detail_v2_status` for details.
+dateCreated    string  ISO 8601 date of when the feed import was started.
+error          string  the error affecting the whole feed import, as opposed to ``errors`` containing errors affecting individual ads. If your feed is in status ``REJECTED`` this should have the reason.
+totalCount     int     the total number of ads found in the feed file.
+okCount        int     the number of ads processed without errors.
+warningCount   int     the number of ads processed with warnings.
+errorCount     int     the number of ads rejected due to errors.
+warnings       object  contains count and a subset of vendorIDs (max 100) of ads having ``warning`` messages, see :ref:`message_struct`.
+errors         object  contains count and a subset of vendorIDs (max 100) of ads having ``error`` messages, see :ref:`message_struct`.
+=============  ======= ====================================
+
+
+.. _message_struct:
+
+Error / Warning messages
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+===============    ============ ====================================
+Field              Type         Description
+===============    ============ ====================================
+count              int          total number of ads affected by this error/warning message
+vendorIdsSample    list<string> subset (max 100) of vendorIDs that were affected by this error
+===============    ============ ====================================
+
+It is not possible to fetch all affected ad ids.
+
+.. _get_feed_import_id_detail_v2_status:
+
+Import Status
+~~~~~~~~~~~~~
+
+============    =================================================
+Status          Description
+============    =================================================
+PENDING         the feed import is currently processing.
+DONE            the feed import has completed successfully. Individual ads could still have failed updating, but the process completed.
+REJECTED        the feed import failed; either the feed file could not be downloaded or the feed file didn't pass validation.
```

Comment:
None of the ads (records?) get processed in such case right?

Reply:
correct. Should I explicitly mention that here too?